### ISSUE TITLE
Jobs app only showing WarmUp function

### ISSUE THREAD

**Issue by frasermclean** [frasermclean/adoptrix#95]
There appears to an issue with the jobs app. The only function that is showing is a "WarmUp" function:

![image](https://github.com/user-attachments/assets/7e9f03ba-c16f-433b-8ef1-4ac0da355657)

This might be related to this issue: 
https://github.com/Azure/azure-functions-host/issues/10169

**Comment by frasermclean** [comment]
This issue is still ongoing an looks like it might be related to GitHub Action deployment issue: https://github.com/Azure/functions-action/issues/245

**Comment by davidobrien1985** [comment]
Yeah, definitely still an issue. No matter how often I deploy, none of my Functions show up.

**Comment by frasermclean** [comment]
> Yeah, definitely still an issue. No matter how often I deploy, none of my Functions show up.

Have you tried deploying from VSCode or CLI as a workaround?

**Comment by lukaferlez** [comment]
Same issue here, makes the flex plan unusable.

**Comment by lukaferlez** [comment]
The issue for us was using ConfigureFunctionsWebApplication() to configure HostBuilder. When we changed to ConfigureFunctionsWorkerDefaults, the functions loaded no problem.

**Comment by im-samz** [comment]
@frasermclean, @davidobrien1985, did y'all end up trying what @lukaferlez did? The Functions team is also trying to root cause why only the WarmUp function showed up.
